src/main/java/net/dv8tion/jda/core/requests/RestAction.java

```diff
@@ -161,13 +162,31 @@
             LOG.error("RestAction queue returned failure: [{}] {}", t.getClass().getSimpleName(), t.getMessage());
         }
     };
+    protected static boolean passContext = false;
```

Comment:
This will change functionality across all running instances of JDA.
Do we want that? Or do we want to provide per-JDA-instance support?

Reply:
I'm also more a fan of making this a setting in JDABuilder

Reply:
I'm unsure why this should be a per-JDA setting but DEFAULT_FAILURE and DEFAULT_SUCCESS should not

Reply:
🤔 That is a good point. We should make a decision now as to whether we make both per-JDA, or both Global.

Reply:
If we want to merge this, we will have to decide on something.